### ISSUE TITLE
Commands for troubleshooting 

### ISSUE THREAD

**Issue by stehx26** [FuelRats/pipsqueak#18]
For example

!xnat

<user>: please can you tap on xbox button, tap left on d pad and scroll down to settings, choose all setting and then choose network. Confirm NAT type is open. 

!xrelog
<user>: please restart the game client by press xbox button, pressing the start button and choosing quit. 

!pcrelog

<user>: please completely quit the game by choosing save and exit to desktop

!xreset

<user> Please turn your xbox off, unplug power, then plug back in and restart. 

etc etc :)

**Comment by duk3luk3** [comment]
These instructions should be in an article on fuelrats.com along with an explanation of the issues that they are supposed to solve. The macros are rather useless without an explanation of what they're for.

**Comment by rwaltr** [comment]
Any situation that reaches this point is likely a _very "case by case"_ topic. At this point we are relying on both the technological skills of both the **Dispatch** and **Client**. 

Devil's Advocate: While I'm not against adding these facts. I do not think they will be used much, **Especially** with the **diversity of consumer technology** relation to routers, port forwarding, UPNP, Internet providers restrictions, will make the endmost troubleshooting a very _non factable_ situation.

### **TLDR:** Not entirely sure how helpful these would actually be.

**Comment by duk3luk3** [comment]
> Devil's Advocate: While I'm not against adding these facts. I do not
> think they will be used much, Especially with the diversity of consumer
> technology relation to routers, port forwarding, UPNP, Internet providers
> restrictions, will make the endmost troubleshooting a very non factable
> situation.

I also see a danger of them being used by people who did not take the time
to learn about all that. Thus why I think there should be a detailed
article about it somewhere.

**Comment by stehx26** [comment]
I'll write an article; because yeah its a good idea. 

I accept the point that its a case by case thing... but is there a difference betwen a PC based dispatcher copy pasting the command procedure from an article or typing !xnat?

The article should (and hopefully will) explain the context and the reasons why one should give those commands for sure; but I dont see it as an argument for not making the commands availible for a dispatcher to use. 

I'll write up the article, then you can see if its relevent to add macros.

**Comment by dewiniaid** [comment]
More things that #49 and friends will allow, since facts could be added on the fly.  (Current implementation requires halfop or better on channel to do it, but it's usually not too hard to find one of those.)

**Comment by Marenthyu** [comment]
We have an article on the website now as well as the facts that can be added on the fly. We temporarily had !xproblems in DrillSqueak but that got lost. Maybe will readd it if the need arises, but it doesnt seems too too necessary.
